webserver/webserver/src/main/java/io/helidon/webserver/http/ServerResponseBase.java

```diff
@@ -187,6 +190,16 @@ public boolean isNexted() {
         return nexted;
     }
 
+    @Override
+    public ServerResponse beforeTrailers(Consumer<ServerResponseTrailers> beforeTrailers) {
+        this.beforeTrailers = beforeTrailers;
+        return this;
+    }
+
+    protected Consumer<ServerResponseTrailers> beforeTrailers() {
```

Comment:
Missing javadoc on a protected method of a public class.

Reply:
Fixed